### ISSUE TITLE
"High latency on CRC GET request" with 1.7.1 (on Glitch)

### ISSUE THREAD

**Issue by shiffman** [twitterdev/autohook#34]
I am running the [glitch autohook example](https://glitch.com/edit/#!/twitter-autohook-tutorial), thank you so much for such a nice concise example!

### Expected behavior

Using `1.2.1` the example runs perfectly, and I see the following output:

```
Removing webhooks…
Removing https://myapp.glitch.me/webhook…
Registering https://myapp.glitch.me/webhook as a new webhook…
Webhook created.
```

However, when I upgrade the package to `1.7.1` I consistently get the following error when trying to create the webhook.

### Actual behavior

```
AuthenticationError: High latency on CRC GET request. Your webhook should respond in less than 3 seconds. (HTTP status: 400, Twitter code: 214)
    at tryError (/rbd/pnpm-volume/fccb6299-90f3-4bdf-b4a6-46d776e1bbf8/node_modules/.registry.npmjs.org/twitter-autohook/1.7.1/node_modules/twitter-autohook/errors/index.js:53:12)
    at Autohook.setWebhook (/rbd/pnpm-volume/fccb6299-90f3-4bdf-b4a6-46d776e1bbf8/node_modules/.registry.npmjs.org/twitter-autohook/1.7.1/node_modules/twitter-autohook/index.js:205:19)
    at processTicksAndRejections (internal/process/task_queues.js:88:5)
    at async Autohook.start (/rbd/pnpm-volume/fccb6299-90f3-4bdf-b4a6-46d776e1bbf8/node_modules/.registry.npmjs.org/twitter-autohook/1.7.1/node_modules/twitter-autohook/index.js:270:7)
```

### Steps to reproduce the behavior

Run the [glitch autohook example](https://glitch.com/edit/#!/twitter-autohook-tutorial) but change `package.json` to the following:

```
{
  "dependencies": {
    "twitter-autohook": "^1.7.1",
    "request": "^2.88.0"
  }
}
```

**Comment by iamdaniele** [comment]
Hi @shiffman, [Glitch blocks incoming requests](https://support.glitch.com/t/solved-403-from-requests-to-stack-overflow/13557/5) that do not have a `User-agent` header. Incoming webhook calls from Twitter do not have a `User-agent` header. Boosted Apps should work as intended.

**Comment by shiffman** [comment]
Thanks for the reply! Unfortunately I am getting this error in a boosted app but let me try testing again to make sure.

**Comment by hkolbeck-streem** [comment]
@shiffman Sorry to bother you, but did you ever figure this out? I'm running into it on 1.7.2 with a boosted project.

**Comment by shiffman** [comment]
@hkolbeck-streem unfortunately no, I'm still running 1.2.1 due to this issue.

**Comment by hkolbeck-streem** [comment]
@shiffman Thanks for the quick response, and sorry to hear it

@iamdaniele Any chance we can re-open this? It seems like it may not be solved as simple as the User-Agent issue, unless I misunderstand something.

**Comment by shiffman** [comment]
Oddly, I did just notice another example bot I created is using 1.7.2 and works deployed to Glitch! I'm not sure why one produces this error and the other does not! The working one is here: https://github.com/CodingTrain/GUMP500-bot (sorry it's not well documented, something I made during a live stream.) It depends on https://github.com/CodingTrain/ChooChooTweets which [has 1.7.2 as a dependancy](https://github.com/CodingTrain/ChooChooTweets/blob/main/package.json). I'll have to investigate this.
